sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -883,6 +895,221 @@ def _export_files(self, bq):
     return table.schema, metadata_list
 
 
+class _CustomBigQueryStorageSourceBase(BoundedSource):
```

Comment:
Thank you for the storage api client support! I am wondering could big query storage API be used to read a temporary table from BQ?

Reply:
Yes it can be used to read temporary tables (those produced as query results).